sources/packages/web/src/components/common/modals/ConfirmExtendTime.vue

```diff
@@ -43,65 +41,30 @@ export default {
       required: true,
       default: "" as ClientIdType,
     },
+    countdown: {
+      type: Number,
+      required: true,
+    },
+    showDialog: {
+      type: Boolean,
+      required: true,
+    },
   },
-  setup(props: any) {
-    const countdown = ref(COUNT_DOWN_TIMER_FOR_LOGOUT);
-    const { showDialog, resolvePromise, showModal } = useModalDialog<boolean>();
-    const interval = ref();
-    const { executeLogout } = useAuth();
-
-    const logoff = async () => {
-      await executeLogout(props.clientIdType);
-    };
+  setup() {
+    const { resolvePromise, showModal } = useModalDialog<boolean>();
 
     const dialogClosed = () => {
```

Comment:
This method and extendTime at line 60 are doing exactly the same except the argument. THey can be unified.